examples/ex12p.cpp

```diff
@@ -2,14 +2,15 @@
 //
 // Compile with: make ex12p
 //
-// Sample runs:  mpirun -np 4 ex12p -m ../data/beam-tri.mesh
-//               mpirun -np 4 ex12p -m ../data/beam-quad.mesh
-//               mpirun -np 4 ex12p -m ../data/beam-tet.mesh -n 10 -o 2 -elast
-//               mpirun -np 4 ex12p -m ../data/beam-hex.mesh -s 3876
-//               mpirun -np 4 ex12p -m ../data/beam-tri.mesh -o 2 -sys
-//               mpirun -np 4 ex12p -m ../data/beam-quad.mesh -n 6 -o 3 -elast
-//               mpirun -np 4 ex12p -m ../data/beam-quad-nurbs.mesh
-//               mpirun -np 4 ex12p -m ../data/beam-hex-nurbs.mesh
+// Sample runs:
+//    mpirun -np 4 ex12p -m ../data/beam-tri.mesh
+//    mpirun -np 4 ex12p -m ../data/beam-quad.mesh
+//    mpirun -np 4 ex12p -m ../data/beam-tet.mesh -n 10 -o 2 -elast
+//    mpirun -np 4 ex12p -m ../data/beam-hex.mesh -s 3876
+//    mpirun -np 4 ex12p -m ../data/beam-tri.mesh -o 2 -sys
+//    mpirun -np 4 ex12p -m ../data/beam-quad.mesh -s 4526 -n 6 -o 3 -elast
```

Comment:
I'd also be OK if we kept the original formatting in this case -- the sample runs don't necessarily have to fit in 80 cols in my opinion.